pkg/trace/stats/quantile/ddsketch_converter.go

```diff
@@ -0,0 +1,180 @@
+package quantile
```

Comment:
`converter.go` should be enough for the file name. Don't forget to add the copyright headers.